### ISSUE TITLE
Deploy New Business Network panel fails if no sample networks available

### ISSUE THREAD

**Issue by bestbeforetoday** [hyperledger/composer#2552]
Composer version: 0.14.3

`import.component.ts` contains this code:
```javascript
onShow(): Promise<void> {
    this.npmInProgress = true;
    return this.sampleBusinessNetworkService.getSampleList()
        .then((sampleNetworkList) => {
            this.sampleNetworks = this.addEmptyNetworkOption(sampleNetworkList);
            this.selectNetwork(this.sampleNetworks[1]);
```
The call to `addEmptyNetworkOption()` inserts an empty network selection to the front of the list of sample networks obtained from `this.sampleBusinessNetworkService.getSampleList()`. It then selects the network at index `1`, which it expects to be the first sample network.

If no sample networks are obtained (so `getSampleList()` returns an empty array), `sampleNetworks[1]` does not exist. In this case the _Deploy New Business Network_ panel greys itself out during display and becomes totally unresponsive. Business networks cannot be deployed.

This occurred running a local (development) Playground at a version level higher than the versions of the sample projects currently published. I suspect this is not likely to happen in the wild, but the code is unnecessarily fragile. The way this error handled is not very graceful either.

**Comment by awjh-ibm** [comment]
Also causes a failure when the versions have not been bumped as all are filtered out

**Comment by sstone1** [comment]
@samjsmith can this be closed of a dupe of #2568 which you're delivering a fix for?

**Comment by samjsmith** [comment]
Yes, this is a dupe of #2568
